spec/requests/api/user_can_see_all_articles_spec.rb

```diff
@@ -9,14 +11,14 @@
       expect(response.status).to eq 200
     end
   end
-  
+
   describe 'GET, No articles been found' do
     before do
-    get '/api/articles'
-  end
+      get '/api/articles'
+    end
 
     it 'No articles been found' do
       expect(response.status).to eq 404
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
There are no changes here. Now, if you format your code properly to start with, you don't have to see these files as part of your PR's

Reply:
We will keep this in mind! Sorry, we started to use solargraph recently in specs files, as it was unable by default.